actions/ingestion/webhook/index.js

```diff
@@ -76,22 +68,27 @@ async function main (params) {
 
     logger.info('Process event data')
     logger.debug(
-          `Process event ${event.type} for entity ${event.entity}`)
+          `Process event ${params.data.event}`)
```

Comment:
is it in a different line on purpose?

Reply:
Good point! I will remove the duplication